purchase_sale_link_by_origin/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+{
+    'name': 'AFS Sale & Purchase',
+    'version': '16.0.1.0.0',
```

Comment:
summary missing?
"Link PO/SO by the PO's Origin in addition to the default behavior that only links them by their lines"